WooCommerce/Classes/ViewRelated/Orders/Simple Payments/SimplePaymentsSummary.swift

```diff
@@ -0,0 +1,256 @@
+
+                    Button(SimplePaymentsSummary.Localization.editNote) {
+                        print("Tapped on Edit")
+                    }
+                    .foregroundColor(Color(.accent))
+                    .bodyStyle()
+                    .renderedIf(content != nil)
+                }
+
+                noteContent()
+
+            }
+            .padding()
+            .background(Color(.listForeground))
+
+            Divider()
+        }
+    }
+
+    /// Builds a button to add a note if no note is present. If there is a note present only displays it
+    ///
+    @ViewBuilder private func noteContent() -> some View {
+        if let content = content {
+
+            Text(content)
+                .bodyStyle()
+
+        } else {
+
+            Button(action: {
+                print("Tapped add note")
+            }, label: {
+                HStack() {
+                    Image(uiImage: .plusImage)
+
+                    Text(SimplePaymentsSummary.Localization.addNote)
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                }
+                .foregroundColor(Color(.accent))
+                .bodyStyle()
+            })
+            .frame(maxWidth: .infinity)
+        }
+    }
+}
+
+/// Represents the bottom take payment button
+///
+private struct TakePaymentSection: View {
+    var body: some View {
+        VStack {
+            Divider()
+
+            // Temporary data
+            Button(SimplePaymentsSummary.Localization.takePayment(total: "$40.0"), action: {
+                print("Take payment pressed")
+            })
+            .buttonStyle(PrimaryButtonStyle())
+            .padding()
+
+        }
+        .background(Color(.listForeground))
+    }
+}
+
+// MARK: Constants
+private extension SimplePaymentsSummary {
+    enum Layout {
+        static let spacerHeight: CGFloat = 16.0
+        static let horizontalStackSpacing: CGFloat = 16.0
+        static let verticalSummarySpacing: CGFloat = 8.0
+        static let verticalNoteSpacing: CGFloat = 22.0
+        static let noSpacing: CGFloat = 0.0
+    }
+
+    enum Localization {
+        static let customAmount = NSLocalizedString("Custom Amount",
+                                                    comment: "Title text of the row that shows the provided amount when creating a simple payment")
+        static let email = NSLocalizedString("Email",
+                                             comment: "Title text of the row that holds the provided email when creating a simple payment")
+        static let emailPlaceHolder = NSLocalizedString("Enter Email",
+                                                        comment: "Placeholder of the row that holds the provided email when creating a simple payment")
+        static let payment = NSLocalizedString("Payment",
+                                               comment: "Title text of the row that shows that list the payment when creating a simple payment")
```

Comment:
nit: Small typo in this comment

```suggestion
                                               comment: "Title text of the row that shows the payment when creating a simple payment")
```